heartbeat/monitors/util.go

```diff
@@ -13,20 +12,23 @@ import (
 )
 
 type funcJob struct {
-	name, typ string
+	name string
 	funcTask
 }
 
 type funcTask struct {
 	run func() (common.MapStr, []TaskRunner, error)
 }
 
+// IPSettings provides common configuration settings for IP resolution and ping
+// mode.
 type IPSettings struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function MakeSimpleJob should have comment or be unexported